srcpkgs/fwupd/template

```diff
@@ -1,23 +1,21 @@
 # Template file for 'fwupd'
 pkgname=fwupd
-version=1.5.8
+version=1.6.3
 revision=1
 build_style=meson
 build_helper="gir"
 # tests require unpackaged umockdev
 configure_args="-Dsupported_build=true
- -Dconsolekit=false -Dgtkdoc=true -Dintrospection=true
+ -Dconsolekit=false -Dintrospection=true
  -Dsystemd=false -Dplugin_altos=false -Dtests=false
- -Dpkcs7=false -Db_lto=false -Defi-includedir=${XBPS_CROSS_BASE}/usr/include/efi
- -Defi-ldsdir=${XBPS_CROSS_BASE}/usr/lib -Defi-ld=${XBPS_CROSS_BASE}/bin/ld
- -Defi-objcopy=${XBPS_CROSS_BASE}/bin/objcopy
- -Defi-libdir=${XBPS_CROSS_BASE}/usr/lib -Delogind=true"
-hostmakedepends="dejavu-fonts-ttf gnutls-tools help2man pkg-config gcab
+ -Db_lto=false -Delogind=true"
+# -Dgtkdoc=true -Dpkcs7=false
```

Comment:
did you leave this comment here by accident?

Reply:
Yes.